core/modules/czar/python/app.py

```diff
@@ -340,7 +339,8 @@ def __init__(self, query, context, setSessionId, resultName=""):
         pass
 
     def _reportError(self, chunkId, code, message):
-        logger.dbg("reporting", chunkId, code, message)
+        # TODO remove lsst/log wrapper
+        logger.dbg("Reporting:\n\tchunkId=[%s]\n\tcode=[%s]\n\tmessage=[%s]\n" % (chunkId, code, message))
```

Comment:
Ok I set a oneliner message.

No, logger wrapper doesn't support lazy formatting, it only join parameter using ' ' as a separator. FYI I opened DM-3037 to remove this wrapper.